src/scss/footer.scss

```diff
@@ -0,0 +1,121 @@
+footer {
```

Comment:
You will probably need to add a z-index to the footer that is higher that the one on parallax items so the parallax items go under the footer when scrolling